src/compiler/types.ts

```diff
@@ -559,56 +552,66 @@ namespace ts {
         | SyntaxKind.FalseKeyword
         | SyntaxKind.FinallyKeyword
         | SyntaxKind.ForKeyword
-        | SyntaxKind.FromKeyword
         | SyntaxKind.FunctionKeyword
-        | SyntaxKind.GetKeyword
-        | SyntaxKind.GlobalKeyword
         | SyntaxKind.IfKeyword
-        | SyntaxKind.ImplementsKeyword
         | SyntaxKind.ImportKeyword
-        | SyntaxKind.InferKeyword
         | SyntaxKind.InKeyword
         | SyntaxKind.InstanceOfKeyword
+        | SyntaxKind.NewKeyword

```

Comment:
why does the member order change here? It doesn't make a difference in unions.

Reply:
Yep, as I said in description `sort KeywordSyntaxKind, so its order is ascending order rather than dictionary order.`.
This change keep consistant with other places, and is much more easier to check whether there is missing type or not.